frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -35,15 +35,26 @@ const DisabledItem = ({ name }: { name: string }) => (
 const QuickLinksAndSettingsBtn = ({ linkGroups }) => {
   return (
     <>
-      <Grid container direction="row" style={{ padding: "8px", justifyContent: "flex-end" }}>
+      <Grid
+        container
+        direction="row"
+        style={{
+          padding: "8px",
+          justifyContent: "flex-end",
+          display: "flex",
+          alignItems: "center",
```

Comment:
Grid should provide props for justify and align and flex should be set automatically.

Reply:
when I remove them I get this issue (even with adding justify="flex-end" for the grid component
![2](https://user-images.githubusercontent.com/66325812/166502637-7edc528a-7be7-46f4-90f6-0295ad5c8e44.gif)

Reply:
oh nvm I get it. I'll fix it now